src/components/animated/_splash.less

```diff
@@ -0,0 +1,74 @@
+.splash-btn {
+  width: 220px;
+  height: 50px;
+  border: none;
+  outline: none;
+  color: #fff;
+  cursor: pointer;
+  position: relative;
+  z-index: 0;
+  border-radius: 10px;
```

Comment:
Can we make the border radius only for the buttons that don't have class `rounded-btn`? I think this is the only change we need to make. The rest looks good

Reply:
Done!